roles/marmotta/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
-JETTY_VERSION: 8.1.16.v20140903
+jetty_ver: 8.1.16.v20140903
```

Comment:
Lowercasing this was also for consistency, though I'm fine with uppercasing everything else instead, if it helps readability.